docs/node.md

```diff
@@ -125,3 +131,9 @@ A Kubernetes administrator typically uses `kubectl` to manage `Node`. Similar
 to Node Controller, `kubectl` command only creates/deletes node representation.
 Note if Kubernetes is running on cloud provider, `kubectl create` a node will
 be refused if Node Controller has already synchronized nodes from cloud provider.
+Admin can choose to deactivate the node using `kubectl`. Deactivating the node
+will not affect any existing pods on the node but it will disable creation of
+any new pods on the node. Node deactivate example:
+```
+kubectl update nodes 10.1.2.3 --patch='{"apiVersion": "v1beta1", "deactivate": true}'
```

Comment:
since it only blocks scheduling, maybe call it "stopScheduling" or something like that? deactivate seems ambiguous, and might be interpreted as also killing the running pods